api_server/migrations/cryoetdataportal/1722635643562_alter_comments/up.sql

```diff
@@ -0,0 +1,74 @@
+comment on table "public"."datasets" is E'Metadata for a dataset';
+comment on column "public"."datasets"."cell_component_id" is E'If the dataset focuses on a specific part of a cell, the subset is included here';
+comment on column "public"."datasets"."cell_strain_id" is E'Link to more information about the cell strain';
+comment on column "public"."datasets"."related_database_links" is E'If a CryoET dataset is also deposited into another database, e.g. EMPIAR, enter the database identifier here (e.g.https://www.ebi.ac.uk/empiar/EMPIAR-12345/). Use a comma to separate multiple links.';
+
+comment on table "public"."dataset_authors" is E'Metadata for authors of a dataset';
+comment on column "public"."dataset_authors"."dataset_id" is E'Numeric identifier for the dataset this author corresponds to';
+comment on column "public"."dataset_authors"."id" is E'A numeric identifier for this author';
+comment on column "public"."dataset_authors"."primary_author_status" is E'Indicating whether an author is the main person associated with the corresponding dataset';
+
+comment on table "public"."dataset_funding" is E'Metadata for a dataset''s funding sources';
+comment on column "public"."dataset_funding"."dataset_id" is E'Numeric identifier for the dataset this funding source corresponds to';
+comment on column "public"."dataset_funding"."id" is E'A numeric identifier for this funding record';
+
+comment on table "public"."runs" is E'Metadata for an experiment run';
+comment on column "public"."runs"."https_prefix" is E'The HTTPS directory path where this dataset is contained';
+comment on column "public"."runs"."name" is E'Short name for this experiment run';
+
+comment on table "public"."tomogram_voxel_spacings" is E'Metadata for a set of tomograms and annotations of a given voxel spacing';
```

Comment:
What are your thoughts on "Metadata for a given voxel spacing related to tomograms/annotations"?

Reply:
Works for me. I just expanded the `/` to ` and `.

Push incoming.